lib/python/qmk/path.py

```diff
@@ -13,23 +14,24 @@ def keymap(keyboard):
         keyboard
             The name of the keyboard. Example: clueboard/66/rev3
     """
-    for directory in ['.', '..', '../..', '../../..', '../../../..', '../../../../..']:
-        basepath = os.path.normpath(os.path.join('keyboards', keyboard, directory, 'keymaps'))
+    keyboard_folder = Path('keyboards') / keyboard
 
-        if os.path.exists(basepath):
-            return basepath
+    for i in range(5):
```

Comment:
This `5` feels like a magic number here. Why 5?
This code is walking up the directory tree, right? I think a `while keyboard_folder != Path('keyboards')` would be more readable.

Reply:
That's the number of subfolders we support when building keyboards.

Reply:
What do you think about `while keyboard_folder != Path('keyboards')` so we don't have to hardcode a number?